app/views/catalog/_document_default.atom.builder

```diff
@@ -3,7 +3,7 @@ xml.entry do
   xml.title presenter(document).render_document_index_label(document_show_link_field(document))
   
   # updated is required, for now we'll just set it to now, sorry
-  xml.updated Time.now.strftime("%Y-%m-%dT%H:%M:%SZ")
+  xml.updated Time.zone.now.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Good catch, I didn't look closely at the `strftime` format. It seems like we should just use `Time#iso8601` to get this right..